### DISCUSSION TITLE
How do I revert to a preivous commit, Id like to go back to 2.3.1

### DISCUSSION THREAD

**Discussion by yovsac** [lllyasviel/Fooocus#3688] [Q&A] [answered]
Could you  please tell me how I can revert to a previous version of Fooocus? 2.3.1

**Comment by Rogala** [comment, marked as answer]
[Download Previous Version](https://github.com/lllyasviel/Fooocus/discussions/1405)

**Comment by Kokunekosan** [comment]
you'll also have to disable automatic update in the .ini file
